fix rescue not executing in includes

dcodix opened the pull request:
Rescue not executed in case in the main playbook there is only an include.
This seems to solve it.

Reference: #14324

Commit:
Referencing https://github.com/ansible/ansible/issues/14324
Rescue not executed in case in the main playbook there is only an include.
This seems to solve it.

jimi-c commented:
See my comment here: https://github.com/ansible/ansible/issues/14324#issuecomment-198759916

Commit:
Small hack to solve a bug on ansible.
The error:
    May 10 2016 08:44:35 - FAILED- 172.16.1.141 - {"msg": "failed to resolve remote temporary directory from ansible-tmp-1462869875.25-154854907397040: `( umask 22 && mkdir -p \"` echo $HOME/.ansible/tmp/ansible-tmp-1462869875.25-154854907397040 `\" && echo \"` echo $HOME/.ansible/tmp/ansible-tmp-1462869875.25-154854907397040 `\" )` returned empty string", "failed": true}

    It is not quite clear why it is happenging. It seems is a mix of things, contemplating ssh+sudo but also some peoople belives that it happens only since kernel 4.X.

jimi-c commented:
Hi @dcodix, following up on old issues and I see I believe this issue was fixed, however you never responded nor gave an example to reproduce the issue you were seeing. If you are still seeing this problem, please let us know, and we will close this issue for now.

If you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:

   * https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
   * https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!
